content/docs/reactivesearch/v3/search/categorysearch.md

```diff
@@ -126,6 +126,12 @@ Example uses:
     > Note:
     >
     > This property only works with [ReactiveSearch API](/docs/search/reactivesearch-api/) i.e when `enableAppbase` is set to `true` in `ReactiveBase` component.
+
+-   **enableQuerySuggestions** `bool` [optional]
+    Defaults to `false`. When enabled, it can be useful to curate search suggestions based on actual search queries that your users are making. Read more about it over [here](/docs/analytics/query-suggestions/).
+
+    > Query Suggestions only work when `enableAppbase` prop is `true`.
```

Comment:
Use 
`> Note: 

Query Suggestions only work when `enableAppbase` prop is `true`.
`
Consistency issue.